.github/pull_request_template.md

```diff
@@ -23,7 +23,8 @@ Figma file: [if it applies]
 ### 👀 Component checklist
 
 - [ ] Percy was checked for any visual regression
-- [ ] A11y tests have been run locally
+- [ ] A11y tests have been run locally (`yarn test:a11y --filter="COMPONENT-NAME
```

Comment:
seems the command is incomplete

Reply:
there was something weird that happened with the formatting; I think it's fixed now.